src/main/java/de/kittybot/kittybot/database/SQL.java

```diff
@@ -44,7 +47,7 @@ public static void createTable(String table){
 		try{
 			String sql = IOUtils.toString(SQL.class.getClassLoader().getResourceAsStream("sql_tables/" + table + ".sql"), StandardCharsets.UTF_8.name());
 			LOG.info("Read sql table from resources: {}", table);
-			getConnection().createStatement().execute(sql);
+			dataSource.getConnection().createStatement().execute(sql);
```

Comment:
what about closing the connection here?

Reply:
yep yep yep yep yep yep yep yep yep yep yep yep yep